gluoncv/torch/model_zoo/action_recognition/i3d_slow.py

```diff
@@ -15,8 +15,7 @@
 __all__ = ['ResNet_SlowFast', 'i3d_slow_resnet50_f32s2_kinetics400',
            'i3d_slow_resnet50_f16s4_kinetics400', 'i3d_slow_resnet50_f8s8_kinetics400',
            'i3d_slow_resnet101_f32s2_kinetics400', 'i3d_slow_resnet101_f16s4_kinetics400',
-           'i3d_slow_resnet101_f8s8_kinetics400', 'i3d_slow_resnet50_f32s2_custom',
-           'i3d_slow_resnet101_f16s4_kinetics700']
```

Comment:
Can you recover `i3d_slow_resnet101_f16s4_kinetics700 `? We need it for this release.